armi/reactor/blueprints/componentBlueprint.py

```diff
@@ -144,6 +160,8 @@ def shape(self, shape):  # pylint: disable=no-self-use; reason=yamlize requireme
     orientation = yamlize.Attribute(type=str, default=None)
     mergeWith = yamlize.Attribute(type=str, default=None)
     area = yamlize.Attribute(type=float, default=None)
+    particleFuelSpec = yamlize.Attribute(type=str, default=None)
+    particleFuelPackingFraction = yamlize.Attribute(type=float, default=None)
```

Comment:
I'm trying to think of a way we could make these attributes on a subclass that is specialized for particle fuel. Still thinking about it though.